### DISCUSSION TITLE
Budibase 0.2.5 released

### DISCUSSION THREAD

**Discussion by shogunpurple** [Budibase/budibase#761] [Show and tell]
budibase 0.2.5 has been released! In this version:
- Help button so users can be pointed towards GH discussions
- Stability improvements around app deployment
- Fixed bug with several dropzone components
- Improved couchdb cookie based authentication
- Backend UI Table text overflow fixes
- UI updates to deployment history to provide richer loading feedback and change detection

**Comment by sardamit** [comment]
Thank you, and congratulations.

Love the simplicity and clean UI.

Now time to wrap my head around it!
Any quick videos to get started?

**Reply by joebudi** [reply]
Thank you @sardamit . I am creating a tutorial today. It would be great if you could test it for me when it's complete?

**Reply by sardamit** [reply]
I absolutely will.
I just tweeted asking if I should record my playtime with Budibase as I struggle (or not) to find my way around the platform! :D

**Reply by joebudi** [reply]
Haha now that's truly testing the UI/UX and platform. Maybe hold off until the next release. We have some nice UI changes coming which make your life a lot easier!

**Reply by sardamit** [reply]
Great, Joe.

By the way, I am using the Community feature on Github for the first time. Any plans on using a platform like Discourse in the future?

**Reply by joebudi** [reply]
Hey. We actually moved from Discourse to Github discussions (which is still in beta). We feel this is the future, and have obvious benefits such as:

It's positioned alongside our roadmap and projects
It's positioned beside the code
Discussions can be transformed into issues

It's been great so far!

**Reply by sardamit** [reply]
In that case, looking forward to explore a new community platform. :)
